Loaded module: cuda/11.8
python: can't open file '/zhome/e5/d/167772/Desktop/02613/02613_HPC_project/Python_scripts/original_script.py': [Errno 2] No such file or directory

real	0m0.028s
user	0m0.017s
sys	0m0.005s
